src/reconcile.rs

```diff
@@ -0,0 +1,81 @@
+use crate::client::Client;
+use crate::error::Error;
+use k8s_openapi::api::core::v1::Pod;
+use kube::api::{ListParams, Meta, ObjectMeta};
+use kube_runtime::controller::ReconcilerAction;
+use std::time::Duration;
+
+pub type ReconcileResult<E> = std::result::Result<ReconcileFunctionAction, E>;
+
+pub fn create_requeuing_reconciler_action(secs: u64) -> ReconcilerAction {
```

Comment:
Just an idea, maybe changing this to 
pub fn create_requeuing_reconciler_action<T>(time: T) -> ReconcilerAction where T: Into<Duration>
{
    ReconcilerAction {
        requeue_after: Some(Duration::from(time)),
    }
}
would make it more versatile, there is one place in controller.rs where this would come in handy (I've commented there as well).

Reply:
This didn't work. I didn't investigate too deply so I just take a `Duration` now.

Reply:
Fine for me. I just typed this out and was happy when Intellij didn't complain about anything, no compiler ever saw this.